pycti/entities/opencti_case_incident.py

```diff
@@ -441,14 +435,14 @@ def contains_stix_object_or_stix_relationship(self, **kwargs):
         else:
             self.opencti.log(
                 "error",
-                "[opencti_case] Missing parameters: id or stixObjectOrStixRelationshipId",
+                "[opencti_caseIncident] Missing parameters: id or stixObjectOrStixRelationshipId",
             )
 
     """
-        Create a Case object
+        Create a Case Incident object
 
-        :param name: the name of the Case
-        :return Case object
+        :param name: the name of the Case Incident
+        :return Feedback object
```

Comment:
`Feedback` ? I guess it should be `Case Incident`

Reply:
Yep it should ! thank you